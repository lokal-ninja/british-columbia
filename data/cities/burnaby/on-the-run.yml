id: 9078341254
type: node
latitude: 49.22157
longitude: -122.99802
postcode: V5J 1B8
city: Burnaby
street: Imperial Street
housenumber: "4692"
phone: +1-604-412-9839
opening_hours: 24/7
website: https://www.ontherunstores.com/locations/43104
