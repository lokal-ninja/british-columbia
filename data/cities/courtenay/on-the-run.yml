id: 223779126
type: node
latitude: 49.67785
longitude: -124.98304
postcode: V9N 2L6
city: Courtenay
street: Cliffe Avenue
housenumber: "2350"
phone: +1-250-334-2811
opening_hours: ""
website: ""
