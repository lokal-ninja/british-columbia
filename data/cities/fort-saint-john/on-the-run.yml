id: 532717886
type: node
latitude: 56.24263
longitude: -120.86166
postcode: V1J 5T5
city: Fort Saint John
street: Alaska Road North
housenumber: "10812"
phone: +1-250-785-8828
opening_hours: 24/7
website: https://www.ontherunstores.com/locations/40201
