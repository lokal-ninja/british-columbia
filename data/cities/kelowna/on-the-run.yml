id: 12179619851
type: node
latitude: 49.91846
longitude: -119.3882
postcode: ""
city: Kelowna
street: Adams Road
housenumber: "105"
phone: +1 250-491-1882
opening_hours: 24/7
website: https://www.ontherunstores.com/locations/43234
