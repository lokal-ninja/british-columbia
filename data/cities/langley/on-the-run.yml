id: 2795776892
type: node
latitude: 49.11719
longitude: -122.66915
postcode: ""
city: Langley
street: "200 Street"
housenumber: 6295
phone: ""
opening_hours: ""
website: ""