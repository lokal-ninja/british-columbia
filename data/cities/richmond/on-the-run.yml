id: 10889324455
type: node
latitude: 49.16748
longitude: -123.13615
postcode: ""
city: Richmond
street: Number 3 Road
housenumber: "6386"
phone: ""
opening_hours: ""
website: ""
