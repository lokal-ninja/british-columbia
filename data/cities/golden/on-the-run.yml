id: 144459626
type: node
latitude: 51.31421
longitude: -116.97349
postcode: V0A 1H0
city: Golden
street: Trans-Canada Highway
housenumber: "1209"
phone: +1-250-344-5151
opening_hours: ""
website: ""
